docs-src/docs/dataset-exploration/prediction-format.md

```diff
@@ -2,7 +2,7 @@
 title: The prediction format
 ---
 
-Predictions use the OpenLabel format/schema. This is the same format as the one used
+Predictions use the OpenLabel format, which is a JSON schema. This is the same format as the one used
```

Comment:
I think OpenLabel is a definition, not a schema. But they do provide a schema which can be used to validate a json :D